.travis.yml

```diff
@@ -1,21 +1,28 @@
-language: scala
-jdk: oraclejdk8
 sudo: required
 
-cache:
-  directories:
-    - $HOME/.ivy2
-    - $HOME/.sbt
-
-services:
-  - docker
+language: scala
+jdk: oraclejdk8
 
 scala:
   - 2.10.6
   - 2.11.7
 
+before_install:
+  - openssl aes-256-cbc -K $encrypted_e4898ca93742_key -iv $encrypted_e4898ca93742_iv -in secrets.tgz.enc -out secrets.tgz -d
+  - tar zxf secrets.tgz
+
 before_script:
-  - sbt update # update dependencies here in order to cache them and mount them later to docker container
+  - sudo curl -fsSL https://get.docker.com/ | sh
+  - sudo docker run --name riak-ts -d -p 8087:8087 -p 8098:8098 basho/riak-ts
```

Comment:
FWIW The 2.10 and 2.11 builds are currently running in parallel (I think that's how Travis does it). So the build time now is a reflection of two builds. We can look at other tricks maybe to get that time down like reusing cache (I had to take that out because of permissions issues with the build). We might be able to tweak this even more to get the time down.